Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md

```diff
@@ -32,7 +28,7 @@ SELECT
     MEM_SIZE,
     MULTICOPY_SIZE,
     COMMIT_SIZE-MEM_SIZE-MULTICOPY_SIZE                    PHANTOM_SIZE,
-    CAST((local.PHANTOM_SIZE)*100/COMMIT_SIZE AS DEC(7,3)) "PHANTOM_%"
+    CAST((local.PHANTOM_SIZE)*100/(MEM_SIZE+MULTICOPY_SIZE) AS DEC(7,3)) "PHANTOM_%"
```

Comment:
The query is the one from https://exasol.atlassian.net/browse/EXASOL-2335 , is this correct?

Reply:
As far as I can see the query to be changed (fixed) is the same as in that roadmap ticket.